.nvmrc

```diff
@@ -1 +1 @@
-14
+v14.19.1
```

Comment:
.nvmrc allows anyone to easily enter the project and use the exact same Node version as us. I'm not sure about best practices, but that seems like enough of a reason to keep it, since it definitely wouldn't be hurting anything.